pkg/log/kzerolog/kzerolog.go

```diff
@@ -0,0 +1,162 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kzerolog
```

Comment:
What does the 'k' represent? Why not just `zerolog`?

Reply:
k for Kubernetes. I don't want to collide with the `zerolog` package from `github.com/rs/zerolog` and have the users use an alias.

Reply:
OK - it just looks a bit strange. Not trying to nitpick but would any user of this package also need to reference `github.com/rs/zerolog`? I imagine they would just call `InitLogging` - after that reference `sigs.k8s.io/controller-runtime/pkg/log`. 

Also for clarity, we could rename `InitLogging` to `InitK8sLogging`.

Reply:
I think it's a good practice anyway not to shadow other package names. Some advanced usages in the future might want to use also `github.com/rs/zerolog` to configure things differently (e.g use JSON logging instead oh text logging)

Accepted the `InitK8sLoggingz suggestion.